3p/integration.js

```diff
@@ -52,7 +53,6 @@ import {
 } from './3p';
 import {startsWith} from '../src/string.js';
 import {urls} from '../src/config';
-import {version} from '../src/internal-version';
 
 // Disable auto-sorting of imports from here on.
 /* eslint-disable sort-imports-es6-autofix/sort-imports-es6 */
```

Comment:
I think you can remove the above two lines, since `eslint-plugin-sort-imports-es6-autofix` now gracefully handles groups of imports, and doesn't mess with the order across groups.

https://github.com/marudor/eslint-plugin-sort-imports-es6-autofix/pull/5

You can test this locally by removing lines 57-58 and running `gulp lint --files 3p/integration.js`. It shouldn't return any errors. If it does, feel free to punt on this.

Reply:
so you mean just move back the sort fix it did? looks like it still error'd when i removed lines 57-58 and ran your command

Reply:
Sorry, let me elaborate. I had to disable the `sort-imports-es6-autofix/sort-imports-es6` rule because it used to mix up groups. Now, it no longer does.

So if you just remove lines 57-58 and run `gulp lint --files 3p/integration.js`, it will flag an error if the imports are out of order.

However, if you remove lines 57-58 and run `gulp lint --files 3p/integration.js --fix`, it should fix any imports that are out of order without moving them across groups.